schemas/dex/SCHEMA.md

```diff
@@ -0,0 +1,186 @@
+| token0_address        | The contract address of token0.                                      | string |
+| token0_amount         | The amount of token0.                                                | number |
+| token1_address        | The contract address of token1.                                      | string |
+| token1_amount         | The amount of token1.                                                | number |
+
+### V2 Transfers
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the transaction.                                    | number |
+| chain_id              | The standard id of the chain.                                        | number |
+| block_number          | The block number of the trade.                                       | number |
+| log_index             | The log index of the event recorded.                                 | number |
+| transaction_hash      | The hash of the transaction.                                         | string |
+| user_address          | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
+| pool_address          | The contract address of the pool.                                    | string |
+| pool_token_balance    | The balance of the pool token, decimal normalized.                   | number |
+
+### V3 Events
+| Property                | Description                                                      | Type   |
+|-------------------------|------------------------------------------------------------------|--------|
+| timestamp               | The timestamp of the transaction.                                    | number |
+| chain_id                | The standard id of the chain.                                        | number |
+| block_number            | The block number of the trade.                                       | number |
+| log_index               | The log index of the event recorded.                                 | number |
+| transaction_hash        | The hash of the transaction.                                         | string |
+| user_address            | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address        | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address           | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
```

Comment:
because we have transfer events, 

we need to replace taker_address into from_address / to_address here so that the transfer events can fit in this table.

take a look at this table : 

```
select * FROM scroll.nuri_lp_activity
```

Reply:
REplaced taker_address with from_address and to_address. We also have transaction_from/to_address which is different from the event level addresses. But this should clear things up.